tests/subsys/debug/coredump/src/main.c

```diff
@@ -26,6 +26,10 @@ void func_1(uint32_t *addr)
 void main(void)
 {
 	printk("Coredump: %s\n", CONFIG_BOARD);
-
+#ifdef CONFIG_BOARD_MIMXRT685_EVK
+	/* for CM33 0x0 is sharememory */
```

Comment:
shared memory?

Reply:
btw, this would probably not work for many other boards, do not want to see things being ifdefed here, we either exclude such platforms or implement this is more scalable way.

Reply:
@nashif , from the CM33 [Technical document](https://developer.arm.com/documentation/100230/0003/functional-description/programmers-model/memory-model?lang=en), 0x0 to 0x1FFFFFFF is code area, but for NXP platform, we separated this ware as
![image](https://user-images.githubusercontent.com/800198/93417316-5f151e80-f8da-11ea-846c-e95f049b7de3.png)

so this is really a platoform dependant, I update the test address to modify the main directly,please check.